tests/Api/executeToggleTaskDoneCommand.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import moment from 'moment';
+import type { App } from 'obsidian';
+import { tasksApiV1 } from '../../src/Api';
+
+jest.mock('obsidian', () => ({
+    Modal: class Mock {},
+}));
+
+window.moment = moment;
+
+const app = {} as App;
```

Comment:
Wow, this is nice. Did I understand correctly that the `app` parameter is not used at all here, so we can pass a `{} as App`?

Why do we need to mock the `Modal` from `obsidian`?

Just curious for science here =)